packages/moonstone/VirtualList/VirtualListBase.js

```diff
@@ -579,11 +588,17 @@ const VirtualListBaseFactory = (type) => {
 				} else {
 					this.isScrolledBy5way = true;
 
-					if (isWrapped && wrap === true) {
+					if (isWrapped) {
```

Comment:
Refactoring suggestion. Thoughts?

```js
					this.isWrappedBy5way = isWrapped;
					if (isWrapped && wrap === true && (
						this.uiRefCurrent.containerRef.current.querySelector(`[data-index='${nextIndex}'].spottable`) == null
					)) {
						this.pause.pause();
						target.blur();
					} else {
						this.focusOnItem(nextIndex);
					}
```

Reply:
I like it - still appears to cover all the bases. Allowing `isWrapped` to set the value for `this.isWrappedBy5way` really allows that bit to be cleaned up nicely :nice: